_posts/2022-08-22-mdakits-intro.md

```diff
@@ -0,0 +1,133 @@
+scientific problems or in some form enhances the MDAnalysis core
+library. An MDAKit can be written by anyone and hosted
+anywhere.
+
+A MDAKit can be **registered** in the MDAKits registry. In this case,
+it has to fulfill a number of additional requirements such as
+open-source licensed, hosted in a version control system, clear
+designation of authors/maintainers, documentation, and tests and
+continuous integration. Registered MDAKits will be *listed publicly*
+and thus be advertised to the whole MDAnalysis community. They will
+also be *continuosly tested* against the latest released version and
+the current development version of the core MDAnalysis library so that
+users and developers have an up-to-date view of the code health of an
+MDAKit.
+
+
+## Why?
+
+The open sharing of code that abides by the basic principles of
+[FAIR](https://doi.org/10.15497/RDA00068) (findability, accessibility,
+interoperability, and reusability) is essential to robust,
+reproducible, and transparent science. However, scientists typically
+are not supported in making the substantial effort required to make
+software FAIR-compliant, or incentivized with academic recognition or
+reward.  
+
+Our goal with MDAKits is to **lower the barrier for researchers to
+produce FAIR software.** 
+
+We support developers in creating new packages, guiding them through
+the process of achieving best practices and FAIR compliance. At the
+same time, we hope to make MDAnalysis useful to a broader community.
+
+
+## How to develop an MDAKit?
+
+We are producing tools for creating MDAKits to help
+developers and we are working on infrastructure to publicize
+MDAKits. Our work on MDAKits is an ongoing process but you can now get
+started creating your own MDAKit:
+
+
+### MDAKit project template
+
+Our first tool is the
+[**cookiecutter-mdakit**](https://github.com/MDAnalysis/cookiecutter-mdakit),
+a [cookiecutter](https://github.com/audreyr/cookiecutter) template
+that generates a **skeleton project** that implements our recommended
+best practices. With [`cookiecutter`
+installed](https://cookiecutter.readthedocs.io/en/latest/installation.html#install-cookiecutter),
+execute the following command inside the folder you want to create the
+skeletal repository
+
+```bash
+cookiecutter gh:MDAnalysis/cookiecutter-mdakit
+```
+
+Follow the prompts or hit enter for the default options. 
+
+Then add your own code to the project. Add tests --- you can extend
+the example tests in the template that are suitable for
+MDAnalysis-based. Commit and push your changes.
+
+
+<small>(The MDAKit cookiecutter is based off the [Cookiecutter for
+Computational Molecular Sciences (CMS) Python
+Packages](https://github.com/MolSSI/cookiecutter-cms) by Levi N. Naden
+and Jessica A. Nash from the [Molecular Sciences Software Institute
+(MolSSI)](http://molssi.org/) and Daniel G. A. Smith of
+[ENTOS](https://www.entos.ai/). Thank you!)</small>
+
+
+### Registering an MDAKit
```

Comment:
We shouldn't advertise this yet please. We can't go into production with what we have currently, it'll just cause chaos.

Reply:
Ok, good that you reviewed ;-)

can you complete your changes—you’re the one who knows best what should be here.